libs/datasets/statistical_areas.py

```diff
@@ -71,7 +71,6 @@ def from_local_public_data() -> "CountyToCBSAAggregator":
         df = pd.read_excel(
             dataset_utils.REPO_ROOT / CBSA_LIST_PATH,
             header=2,
-            convert_float=False,
```

Comment:
[`convert_float`](https://pandas.pydata.org/pandas-docs/version/1.5/reference/api/pandas.read_excel.html) is deprecated. See https://github.com/pandas-dev/pandas/issues/41127 for some more context